u1-request-response/app.js

```diff
@@ -0,0 +1,156 @@
+//localhost:3000
+app.get('/', (req, res) => {
+    res.send('better world')
+  })
+
+// Routes ส่งข้อความเข้าโดยตรงโดยhtml
+//localhost:3000/html
+app.get('/html', (req, res) => {
+    res.send(`
+    <html><body>
+        <h1>Technologies</h1>
+            <ul>
+                <li>JavaScript</li>
+                <li>Node</li>
+                <li>Express</li>
+            </ul>
+    </body></html>
+  `)
+  })
+
+//localhost:3000/Json
+  app.get('/json', (req, res) => {
+    res.json(
+        {
+            technologies: [
+                'javascript',`typescript`, 'Noodle', 'Express'
+            ]
+        })
+  })
+
+//localhost:3000/status
+  app.get(`/status` ,(req,res) => {
+      res.sendStatus(403)
+  })
+
+  app.get(`/status-200` ,(req,res) => {
+    res.sendStatus(200)
+})
+
+app.get(`/status-400` ,(req,res) => {
+    res.sendStatus(400)
+})
+
+    //localhost:3000/status2
+    app.get(`/status2` ,(req,res) => {
+        res.status(500).send(`Oh no,something went wrong`)
+    })
+
+//locahost:3000/html-pit
+    app.get(`/html-pit` ,(req,res) => {
+        res.send(`
+        <html><body>
+            <h1>Hello</h1>
+            <img src="/images/c.jpg" width="300" />
+        </html></body>   
+        `)
+    })
+
+//LabIndex
+//localhost:3000/labs
+    app.get(`/labs` ,(req,res) => {
+        res.send(`
+        <html>
+            <body>
+                <li>1. <a href="/profiles/bank">/profiles/bank</a></li>
+                <li>2. <a href="/api/profiles/bank">/api/profiles/bank</a></li>
+                <li>3. <a href="/cat-simulator">/cat-simulator</a></li>
+                <li>4. <a href="/admin/grades">/admin/grades</a> AND <a href="/api/grades">/api/grades</a> AND <a href="/boredom">/boredom</a></li>
+            </body>
+        </html>
+        `)
+    })    
+
+//localhost:3000/profiles/bank
+    app.get(`/profiles/bank`,(req,res) => {
+        res.send(`
+        <html><body>
+            <h1>Thanawat Samat<h1>
+            <h4> Nickname: Bank <br>
+                Age: 20 <br>
+                Like: color black&blue. 
+                Motto: nothing is impossible
+            <h4>
+        </body></html>
+        `)
+    })
+
+    app.get(`/api/profiles/bank`,(req,res) => {
+        res.send(
+        {
+            "name": "Thanawat Samat",
+            "Nickname": "Bank",
+            "Age": "20",
+            "Like": ["color black&blue",
+                "nature",
+                "socializing"
+            ],
+            "motto": "nothing is impossible"
+        }
+        )
+    })
+
+//localhost:3000/cat-simulator
+    app.get(`/cat-simulator`,(req,res) => {
+        res.send(`
+        <html>
+            <head><link rel="stylesheet" href="/styles.css"></head>
+                <body class="cats">
+                    <a href="/cat-result">Pet the cat</a><br>
+                    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9e/British_shorthair_cat.jpg/800px-British_shorthair_cat.jpg">
```

Comment:
ตรวจสอบให้แน่ใจว่าคุณสามารถเชื่อมโยงไปยังรูปภาพในเครื่องได้เช่นกัน! (ในไดเร็กทอรี `public`)
Make sure that you are comfortable linking to a local image too! (in the `public` directory)